### ISSUE TITLE
Documentation is unusual and hard to follow

### ISSUE THREAD

**Issue by paulbrzeski** [noelboss/featherlight#308]
This plugin has the potential to be very useful but it's hampered by your README. 

I had to google for guides to comprehend how to use featherlight, not ideal considering how long the README is.

Please please please update your documentation.

**Comment by robneu** [comment]
This would probably be more useful if you had actual suggestions for improving it...

**Comment by paulbrzeski** [comment]
It's a bit hard to provide specific feedback when the basis of this issue is that I find the original document difficult to quickly absorb.

I'm not saying it's unusable. You can read it and make sense of it as a developer...

Off the top of my head I'd say:
- the headings and the overall content structure are logical but the sheer size of the document makes it hard to go back and forth (internal anchors might be an idea), 
- a lot of this document has key information embedded into sentences, I can't suggest a one size fits all approach but the original author would obviously understand the content better and might want to think of ways to rephrase, restructure or maybe even just use text boldness to highlight things.
- replace the examples with a set of example code in a folder, github.io or a proxy site are a great way to quickly have viewable demos inside your repo
- restructure the Configuration section to be clearer about how the attributes should look

**Comment by markosjal** [comment]
I would have to say that I AGREE with this. There are few full examples and most of the documentation is code snips which unless one has advanced knowledge the user will not understand full implementation. I have spent WEEKS trying to figure out how to resize the featherlight modal image. When I found the solution by accident the solution was nothing like what I see in the documentation. I even posted a message on Stackexchange as recommended and no answer there either. Now I am trying to figure out why I can not make an iframe of a PDF a link to a modal view of the samel. and again I do not get good examples . In this instance I can use text as in your example and open the modal from a text link but it does not work from an iframe, no matter what I do.

In fact I think featherlight is great however it appears to be for folks in a certain "club" that have access to the documentation, and not really for simple folk like me.

**Comment by marcandre** [comment]
> a certain "club" that have access to the documentation

There is no such thing.

Ultimate documentation is always the source.

I am maintaining the project but spending a minimal amount of time on it.

This is open source, and it is open source not backed by any big company either. Everyone is welcome to make positive changes.

**Comment by markosjal** [comment]
Te comment of "private club" was not intended to be serious, sorry

**Comment by paulbrzeski** [comment]
Sorry if I sounded ungrateful or rude in my op @marcandre , I posted this issue in the best interests of this project. With any luck, an enthusiast with time will submit a pull request and this will sort the issue out :)
